specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscCompilationJob.json

```diff
@@ -0,0 +1,608 @@
+          {
+            "name": "compilationJobName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The the DSC configuration Id."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscCompilationJob"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/compilationjobs": {
+      "get": {
+        "tags": [
+          "DscCompilationJob"
+        ],
+        "operationId": "DscCompilationJob_ListByAutomationAccount",
+        "description": "Retrieve a list of dsc compilation jobs.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/compilationjoboperations"
+        },
+        "x-ms-examples": {
+          "List DSC Compilation job in Automation Account": {
+            "$ref": "./examples/listCompilationJobsByAutomationAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscCompilationJobListResult"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/compilationjobs/{jobId}/streams/": {
```

Comment:
>/ [](start = 177, length = 1)

Is this last slash really needed?

Reply:
Done.

Reply:
It is still there :(

Reply:
oh. It could be because of the merge issues. Fixing it now.

Reply:
Fixed